### PR TITLE
Fix small bugs

### PR DESCRIPTION
**Pull request by HolmesJJ**

### CODE REVIEW

File: src/main/java/seedu/address/model/reservation/Reservation.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public boolean isSameDate(LocalDateTime otherDateTime) {
     }
 
     /**
-     * Returns true if a given data time is after current date time.
+     * Returns true if a given date time is after current date time.
      */
     public static boolean isValidDateTime(DateTime dateTime) {
         LocalDateTime otherDateTime = DateTimeUtil.parseDateTime(dateTime.value);
```
Comment (zzybluebell) [inline src/main/java/seedu/address/model/reservation/Reservation.java:107]: Good

File: docs/AboutUs.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ You can reach us via email, as indicated below.
 <img src="images/zzybluebell.png" width="200px">
 
 [[github@zzybluebell](http://github.com/zzybluebell)]
+[[portfolio@zzybluebell](team/zzybluebell.md)]
```
Comment (zzybluebell) [inline docs/AboutUs.md:42]: thanks awesome

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 3]
```diff
@@ -56,6 +58,13 @@
...
+        getRoot().addEventFilter(KeyEvent.KEY_PRESSED, event -> {
+            System.out.println(event.getCode());
+            if (event.getCode() == KeyCode.ESCAPE) {
+                Stage stage = (Stage) getRoot().getScene().getWindow();
+                stage.close();
+            }
+        });
     }
 
     /**
```
Comment (Morrow1ndy) [inline src/main/java/seedu/address/ui/HelpWindow.java:70]: LGTM!

**Review by Morrow1ndy — COMMENTED** [review]
LGTM

**Pull request merged**